Makefile

```diff
@@ -20,9 +20,13 @@ endif
 
 all: image
 
-image:
+image: make-local
```

Comment:
i would not make a local build a dependency when doing a container build.  please keep docker build and local build separate